src/api/expenses.py

```diff
@@ -19,14 +54,17 @@ def get_expense(user_id: int, expense_id: int):
     - `category`: the user defined category of the item
     - `description`: the user defined description of the item
     """
-    json = None
-
-    with db.engine.connect() as conn:
-        result = conn.execute("").fetchone()
-        if result is None:
-            raise HTTPException(status_code=404, detail="expense not found.")
-
-    return json
+    user = get_user(user_id)
+    expense_id, _, date, cost, category_id, description = get_expense(
+        user[0], expense_id)
```

Comment:
Why is this user[0] and not just user. I'm probably just missing something but I don't get it. Also if this is necessary can we define this 0 as a constant?

Reply:
Oh, I just used ``user[0]`` since it takes in the user_id. You can change ``user[0]`` to ``user_id`` and it would be the same.